### DISCUSSION TITLE
How to Enabling or disabling Amavis for existing installed Modoboa

### DISCUSSION THREAD

**Discussion by shanmugad** [modoboa/modoboa#2735] [Help]
Hi, 

In Modoboa i noticed Spamassassin & Clamav only work when Amavis is installed and enabled. 

How to disable Amavis, spamassassin & Clamav in existing modoboa installed application (not new installation). 

I want to disable it temporary due to i noticed slow performance in receiving and sending.

**Comment by PatTheMav** [comment]
You should be able to comment out the `proxy_filter` options for `smtpd` and `submission` in postfix' `master.cf` file so that neither incoming nor outgoing mails are piped through amavis' instance on port 10024.

You can leave the rest as-is (including the modoboa extension) if you just want to temporarily have emails bypass it.

**Reply by shanmugad** [reply]
What is the path of `master.cf` ?

**Reply by tonioo** [reply]
/etc/postfix/master.cf

**Reply by shanmugad** [reply]
@tonioo @PatTheMav 

Can you please mention which line i have to comment? (line number)

**Reply by PatTheMav** [reply]
I couldn't tell you because I don't know what the default `master.cf` looks like - given that you're doing a bit of open-heart surgery here, it falls upon you to familiarise yourself with the format of that file, what each directive does and then make the informed choice/change to the file (because you need to be able to revert/fix whatever you might break).

If you struggle to understand how the file works (which is fine, it's not the most straight-forward system at first glance) then you shouldn't be making such changes in the first place.

**Reply by shanmugad** [reply]
@PatTheMav I got your point. Just asked you because it you who suggested to edit `master.cf`

I am trying to follow your above Guide but it already look commented.

**Comment by shanmugad** [comment]
@tonioo 

I am confused what to comment to disable `Amavis, spamassassin & Clamav` in `master.cf`

![image](https://user-images.githubusercontent.com/121617479/210789563-0a896970-9b9e-4af9-b0be-298df87a2c5b.png)

And how to comment? `#` or `//` or `/**` ?

It look like it already commented?

**Reply by tonioo** [reply]
@shanmugad you used the right character to comment lines. And I think the communication with Amavis should be disabled with your modifications

**Reply by shanmugad** [reply]
@tonioo  
I have not commented it. It was already in commented with `#`

Is that mean  Amavis, spamassassin & Clamav was disabled by default since installation?
